internal/hcp/api/client.go

```diff
@@ -92,55 +121,62 @@ func (c *Client) loadOrganizationID() error {
 }
 
 func (c *Client) loadProjectID() error {
+	if c.ProjectID != "" {
```

Comment:
Since this is invoked only when the project ID is not empty, do we need to perform this check?